src/python/Operations/Trello/TrelloClient.py

```diff
@@ -0,0 +1,128 @@
+import os
+import json
+import logging
+from logging import Logger
+
+from typing import Optional
+
+
+class TrelloClient(object):
```

Comment:
@amanda-grp I think it's better to place Trello under `Services` package and remove `Operations`. I imagine `Services` as the package which contains all the modules interacting w/ external services. Having an `Operations` package just complicates things and makes it harder to understand. What do you think? 

Having a `Databases` package makes sense to some extent , though. Databases are also external services which we use, but they are special and it makes sense to me to place them in a separate package.

Having a `Storage` package and placing `EOS` there is something that I'm not very sure of.  EOS is also a "Service" after all. Why not just place it `Services`?  What do you think about these organization in general? Does it make sense to you?

Reply:
I think I agree with you about these points — although it still seems reasonable to me to place Trello and JIRA (not migrated yet)  together in a separate package for operational services.

Also, I'm not sure if everything in the `Components` package makes sense together, what do you think?

Reply:
@haozturk EOS and Trello are in the Services package now